charts/ingress-nginx/values.yaml

```
@@ -279,11 +279,11 @@ controller:
   #       - key: app.kubernetes.io/name
   #         operator: In
   #         values:
-  #         - ingress-nginx
+  #         - '{{ include "ingress-nginx.name" . }}'
   #       - key: app.kubernetes.io/instance
   #         operator: In
   #         values:
-  #         - ingress-nginx
+  #         - "{{ .Release.Name }}"
```

```suggestion
  #         - '{{ .Release.Name }}'
```

I'm not sure if double quotes work. I had trouble with them while implementing templatable topology spread constraints.